src/js/tracks/text-track-display.js

```diff
@@ -58,6 +58,19 @@ class TextTrackDisplay extends Component {
         let track = tracks[i];
         this.player_.addRemoteTextTrack(track);
       }
+
+      let modesToShow = {'captions': 1, 'subtitles': 1, 'descriptions': 1};
+      let trackList = this.player_.textTracks();
+
+      if (trackList) {
+        for (let i = 0; i < trackList.length; i++) {
+          let track = trackList[i];
+          if (track.default && track.kind in modesToShow) {
+            track.mode = 'showing';
+            break;
```

Comment:
Might be my misunderstanding of how it works natively, but is this also what happens normally? I.e. If I have both a caption and description having the default attribute, it now matters in which order I list them.